driver-core/src/test/functional/com/mongodb/client/model/AggregatesFunctionalSpecification.groovy

```diff
@@ -291,6 +291,60 @@ class AggregatesFunctionalSpecification extends OperationFunctionalSpecification
         fromHelper?.drop()
     }
 
+    @IgnoreIf({ !serverVersionAtLeast(3, 6) })
+    def '$lookup with pipeline without variables'() {
+        given:
+        def fromCollectionName = 'holidays'
+        def fromCollection = getCollectionHelper(new MongoNamespace(getDatabaseName(), fromCollectionName))
+        def collection = getCollectionHelper()
+
+        collection.drop()
+        fromCollection.drop()
+
+        fromCollection.insertDocuments(
+                Document.parse('{ "_id" : 1, year: 2018, name: "New Years", date: { $date : "2018-01-01T00:00:00Z"} }'),
+                Document.parse('{ "_id" : 2, year: 2018, name: "Pi Day", date: { $date : "2018-03-14T00:00:00Z" } }'),
+                Document.parse('{ "_id" : 3, year: 2018, name: "Ice Cream Day", date: { $date : "2018-07-15T00:00:00Z"} }'),
+                Document.parse('{ "_id" : 4, year: 2017, name: "New Years", date: { $date : "2017-01-01T00:00:00Z" } }'),
+                Document.parse('{ "_id" : 5, year: 2017, name: "Ice Cream Day", date: { $date : "2017-07-16T00:00:00Z" } }')
+        )
+
+        collection.insertDocuments(
+                Document.parse('''{ "_id" : 1, "student" : "Ann Aardvark", 
```

Comment:
The build failure is due to trailing whitespace on this line.  If you run  `./gradlew driver-core:check' it should fail for you too.